cypress.config.ts

```diff
@@ -10,10 +10,12 @@ export default defineConfig({
         pass: "Dog8code",
         git_user: "",
         git_password: "",
-        jira_key: "",
+        jira_url: "https://mta-qe-testing.atlassian.net/",
+        jira_key:
+            "ATATT3xFfGF0WxoR7v-di3daYt8F_xPXPd8XbR1QlBgSYf7bCTmW8u0_NozzOiTyq0xpmNzIMzr6jgwPOvDMCS4lpSsPbtlx4PJDj1L39U5VVZwoaNy7wu1jbf3oRV4MDSjI505Bb-hMSIQQwxCqmahKyWTe_zCPnNf0MduDYzQxzVspEaDTDnk=37A51C8A",
         jira_email: "mta.qe.testing@gmail.com",
         jira_token:
-            "ATATT3xFfGF0OaEyVWcq8N0FfE9ahkpAgC1_z6x_xtojG-51MtinINlMRSxP8rYrvnSK9TaCJSyVZrd5ptB6ajVmmK-vP0vuMFDNEaeb4Hx_WnPtipJxXUsgnr4TKeA99qFulZVophvjBviLjp6HmoM9pqGSJr6uxjpNQr2R3cyMaka7j1rqlU4=420CCF51",
+            "ATATT3xFfGF0WxoR7v-di3daYt8F_xPXPd8XbR1QlBgSYf7bCTmW8u0_NozzOiTyq0xpmNzIMzr6jgwPOvDMCS4lpSsPbtlx4PJDj1L39U5VVZwoaNy7wu1jbf3oRV4MDSjI505Bb-hMSIQQwxCqmahKyWTe_zCPnNf0MduDYzQxzVspEaDTDnk=37A51C8A",
```

Comment:
**issue:** The token is a secret, it shouldn't be defined in the cypress.config. same for the URL, key and email.

**suggestion:**: You can just leave the placeholders. i.e `jira_token: ""`

Reply:
@sshveta asked me to put it here, she said that is not sensitive info. But I definitely can remove it and actually agree it should be removed. In that case I need to update CI to make those tests work in future and it should be added to Jenkins as env variables.

Reply:
Removed

Reply:
Oh ok, then I guess we can leave it there